feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -61,7 +67,8 @@ var (
 	}
 
 	atePort1 = attrs.Attributes{
-		Name:    "atePort1",
+		Name:    "port1",
```

Comment:
Is atePort1 required to have a specific name in OTG?

Reply:
No, but these names have to match the atedut_4.testbed file for ate ports ids, which are currently set to "port1", "port2, ... etc